app/views/layouts/steps_to_become_a_teacher.html.erb

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html lang="en" class="govuk-template">
+    <%= render "sections/head" %>
+    <%= analytics_body_tag class: "govuk-template__body govuk-body", data: { controller: "video link", target: "link.content" }, id: "body" do %>
+    <div id="skiplink-container">
+        <div>
+            <a href="#main-content" class="skiplink govuk-link">Skip to main content</a>
+        </div>
+    </div>
+        <%= render "sections/header" %>
+        <main role="main" id="main-content">
+            <%= render "sections/hero",
+                :header => @front_matter["title"],
+                :image => @front_matter["image"],
+                :mobileimage => @front_matter["mobileimage"],
+                :deep => @front_matter["deepheader"],
+                :subtitle =>  @front_matter["subtitle"],
+                :mailinglist => @front_matter["mailinglist"]
+            %>
+            <section class="content content--<%= @front_matter["direction"] %> <%= @front_matter["fullwidth"] ? "" : "container-1000" %>">
+                <% if @front_matter["on_this_page"].present? %>
```

Comment:
Not sure what `on_this_page` relates to? I can't find it in the content repo at all?

Reply:
It's in the linked branch only, like on [this page](https://github.com/DFE-Digital/get-into-teaching-content/blob/convert-remaining-content-to-markdown/content/returning-to-teaching.md). It's used to generate these boxes for the right column

![Screenshot from 2020-11-09 14-14-52](https://user-images.githubusercontent.com/128088/98552077-f58c0d80-2295-11eb-83ba-bdfda8d8ad3a.png)

Reply:
Ah yeah I see them now, I wonder if calling it something else would be clearer, maybe `jump_links`? also, it may not be worth the effort just now but if we're doing this on a lot of pages I wonder if we can infer them from the headings and just have a flag in the markdown like `render_jump_links: true` or similar?

Reply:
They're currently on 3 pages. It's definitely not the best or most-scalable approach. Kramdown does support tables of contents, but rendering the `toc` outside of the body of the document isn't straightforward.

As there are only three at the moment let's go with the manual approach and it can be something to revisit if we add more.

Reply:
I did rename the entry to `jump_links` though, it's definitely less cryptic.